src/bankAccount.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 class BankAccount {
-  // Write your code here
+  constructor(name, amountOfMoney) {
+    this.name = name;
+    this.amountOfMoney = amountOfMoney;
```

Comment:
```suggestion
    this.amount = amount;
```
make it easy as possible